dask/dataframe/backends.py

```diff
@@ -433,6 +435,18 @@ def _nonempty_series(s, idx=None):
     return out
 
 
+@meta_lib_from_array.register(Array)
+def meta_lib_from_array_da(x):
+    # Use x._meta for dask arrays
+    return meta_lib_from_array(x._meta)
+
+
+@meta_lib_from_array.register(np.ndarray)
+def meta_lib_from_array_numpy(x):
+    # numpy -> pandas
+    return pd
```

Comment:
I think the current approach in this PR is relatively simple. However, it is not clear to me if the dispatch should be to the DataFrame library for `meta`, or to the DataFrame constructor. Any opinions on this @ian-r-rose @jrbourbeau ?